### PR TITLE
webui: added sysNames to pulldowns and the main page

### PR DESCRIPTION
**Pull request by aldemira**
Also made dev dependency list look more like devices page

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by laf** [comment]
So this needs a little update. The devices page does some checks on the hostname to decide what it should show. We should wrap the existing code from `html/includes/table/devices.inc.php` into a function and use it in both places:

```php
        if (format_hostname($device) !== $device['sysName']) {
            $hostname .= '<br />' . $device['sysName'];
        } elseif ($device['hostname'] !== $device['ip']) {
            $hostname .= '<br />' . $device['hostname'];
        }
```

**Comment by aldemira** [comment]
@laf Can you give me a couple weeks to sort it out? I'm a bit too busy nowadays.

**Comment by laf** [comment]
@aldemira Absolutely.

File: html/includes/functions.inc.php [hunk 1]
```diff
@@ -1696,3 +1696,23 @@ function get_zfs_pools($device_id)
...
 }
+
+/**
+ * Returns the sysname of a device with a html line break prepended.
+ * if the device has an empty sysname it will return device's hostname instead
+ * And finally if the device has no hostname it will return an empty string
+ * @param device array
+ * @return string
+ */
+function add_sysname_or_hostname($device)
```
Comment (laf) [inline html/includes/functions.inc.php]: Maybe `return_sysname_or_hostname`?

Comment (murrant) [reply html/includes/functions.inc.php]: How about `get_device_name()`

**Comment by aldemira** [comment]
Thanks guys, I wasn't keen on that function name as well :) 👍

File: html/includes/functions.inc.php (hunk 1 above)
Comment (laf) [reply html/includes/functions.inc.php]: I'm wondering if we pass a $html = false flag to it and it can then be re-used in other places.

Comment (aldemira) [reply html/includes/functions.inc.php]: I removed the html bits altogether.

**Pull request merged**